recipes/pynnotator/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pynnotator" %}
+{% set version = "1.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "4e777813fec487e4352ad4fd6bd55ae5286a120fab848cdba7c52c126645aab5"
+
+build:
+  number: 0
+  entry_points:
+    - pynnotator=pynnotator.main:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python 
+
+requirements:
+  host:
+    - Cython
+    - pip
+    - pysam
+    - python >=3
+    - wheel
+  run:
+    - Cython
+    - pysam
+    - python >=3
+    - wheel
+
+test:
+  imports:
+    - pynnotator
+    - pynnotator.helpers
+    - pynnotator.tests
+  commands:
+    - pynnotator --help
+  requires:
+    - nose
+
+about:
+  home: "http://github.com/raonyguimaraes/pynnotator"
+  license: "BSD"
+  license_family: "BSD"
+  license_file: ""
+  summary: "A Python Annotation Framework for VCFs using multiple tools"
+  doc_url: ""
+  dev_url: ""
```

Comment:
Please remove the `doc_url` and `dev_url` entries. Also check to see if this ships a copy of its license and put that in `license_file` if so (otherwise, remove the license_file entry).

Reply:
Done